packages/odyssey-react-mui/src/ui-shell/UiShell.tsx

```diff
@@ -116,30 +127,63 @@ const UiShell = ({
     };
   }, [onSubscriptionCreated, subscribeToPropChanges]);
 
-  return (
-    <ErrorBoundary fallback={appComponent} onError={onError}>
+  return activeBreakpoint === "none" ? null : (
```

Comment:
nit To be safe, when `activeBreakpoint === "none"`, would it be possible to just render the app content? Just in case `activeBreakpoint` never gets updated from `none`. Not a blocker for this PR

Reply:
It renders the app content. I actually _don't_ like that it does that though because you get a flash of unstyled app content, then the UI Shell suddenly injects itself. It should actually be the other way around.